spring-boot-starter/spring-boot-starter-registry/src/test/java/io/servicecomb/springboot/starter/registry/TestCseAutoConfiguration.java

```diff
@@ -0,0 +1,60 @@
+
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicecomb.springboot.starter.registry;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import io.servicecomb.serviceregistry.RegistryUtils;
+import io.servicecomb.serviceregistry.api.registry.MicroserviceInstance;
+import mockit.Expectations;
+import mockit.Tested;
+
+public class TestCseAutoConfiguration {
+
+  @Tested
+  private CseAutoConfiguration cseAutoConfiguration;
+
+  @Before
+  public void setUp() throws Exception {
+    MicroserviceInstance microserviceInstance = Mockito.mock(MicroserviceInstance.class);
+    new Expectations(RegistryUtils.class) {
+      {
+        RegistryUtils.init();
+        RegistryUtils.run();
+        RegistryUtils.getMicroserviceInstance();
+        result = microserviceInstance;
+
+      }
+    };
+    System.setProperty("cse.rest.address", "127.0.0.1:8081");
+    cseAutoConfiguration = new CseAutoConfiguration();
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    cseAutoConfiguration = null;
+  }
+
+  @Test
+  public void testGetTransport() {
+    Assert.assertNotNull(cseAutoConfiguration);
```

Comment:
It's better to add an assert to test endpoint is the expected one

Reply:
fixed